specification/servicelinker/resource-manager/Microsoft.ServiceLinker/preview/2022-11-01-preview/servicelinker.json

```diff
@@ -0,0 +1,2555 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft.ServiceLinker",
+    "description": "Microsoft.ServiceLinker provider",
+    "version": "2022-11-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
```

Comment:
Please provide a link to the PR from the private repo if this was already reviewed. Also submit any delta between the already reviewed PR and the current one as a separate commit so only those parts may be reviewed.

Reply:
Here are the pr reviewed in the private repo. There is no delta between the already reviewed PR and the current one.
https://github.com/Azure/azure-rest-api-specs-pr/pull/7823
https://github.com/Azure/azure-rest-api-specs-pr/pull/8424
https://github.com/Azure/azure-rest-api-specs-pr/pull/9590
https://github.com/Azure/azure-rest-api-specs-pr/pull/8996